### PR TITLE
Group enh

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

**Review by etokareva — APPROVED** [review]

File: demo/pages/grouping-model/index.js [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
+			context.class('group', {background: '#E3F2FD'});
+		}
+	};
+
+	this.rowOptions = {
+		trigger: 'click',
+		actions: [
+			new qgrid.Action(
+				new qgrid.Command({
+					canExecute: () => {
```
Comment (etokareva) [inline demo/pages/grouping-model/index.js:20]: is it already true by default?

File: src/core/infrastructure/range.d.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export declare class Range {
+	constructor(start: Number, end: Number);
```
Comment (titovmx) [inline src/core/infrastructure/range.d.ts:2]: Do you need Number object type exactly here?

File: src/core/column-type/column.model.view.js [hunk 3]
```diff
@@ -19,8 +19,10 @@ TemplatePath.register('custom-cell-edit', (template, column) => {
 export class ColumnView {
 	constructor(model) {
 		this.model = model;
+
 		this.colspan = 1;
 		this.rowspan = 1;
+		this.index = -1;
```
Comment (titovmx) [inline src/core/column-type/column.model.view.js:25]: Should ```index``` property in ```ColumnModel``` be removed?
Also you should update d.ts files I guess

File: demo/pages/grouping-model/index.js (hunk 1 above)
Comment (klumba12) [reply demo/pages/grouping-model/index.js:20]: this is just copy paste)

**Pull request merged**